src/include/parser/cypher_clause.h

```diff
@@ -39,4 +39,13 @@ Query *cypher_parse_sub_analyze(Node *parseTree,
                                 CommonTableExpr *parentCTE,
                                 bool locked_from_parent,
                                 bool resolve_unknowns);
+
+typedef Query *(*transform_method)(cypher_parsestate *cpstate,
+                                   cypher_clause *clause);
+
+ParseNamespaceItem *transform_cypher_clause_as_subquery(cypher_parsestate *cpstate,
```

Comment:
Same thing goes for this transform_cypher_clause_as_subquery definition as mentioned for the Query typedef. Any particular reason why the declaration was left in the .c file? Readability?